kuma/users/templates/users/user_detail.html

```diff
@@ -1,20 +1,21 @@
 {% extends "base.html" %}
 
-{% set display_name = user_display(profile.user) %}
-{% set username  = profile.user.username %}
-{% set accounts = social_accounts(profile.user) %}
+{% set display_name = user_display(user) %}
+{% set username  = detail_user.username %}
+{% set accounts = social_accounts(user) %}
+{% set is_me = request.user.pk == detail_user.pk %}
```

Comment:
:+1: FWIW I suspect there may have been an existing bug showing the **viewing** user's `login_service_icon`. So some of this confusion was there before. A double-check to make sure we show what's expected may catch that (or another) bug too.